util/compiler.ts

```diff
@@ -2,8 +2,9 @@ import { BN } from 'ethereumjs-util'
 import { IInstruction, IReferenceItem } from 'types'
 
 // Version here: https://github.com/ethereum/solc-bin/blob/gh-pages/bin/list.txt
-export const compilerSemVer = 'v0.8.19'
-export const compilerVersion = `soljson-${compilerSemVer}+commit.7dd6d404`
+// export const compilerSemVer = 'v0.8.19'
+// export const compilerVersion = `soljson-${compilerSemVer}+commit.7dd6d404`
+export const compilerVersion = `soljson-v0.8.21+commit.d9974bed`
```

Comment:
Either keep the concatenated semver + version or keep the new one and remove comments.